Makefile

```diff
@@ -30,7 +30,15 @@ ALL_BINARIES_PLATFORMS= $(addprefix linux/,$(ALL_ARCHITECTURES)) \
 
 # Tools versions
 # --------------
-GOLANGCI_VERSION:=1.64.8
+GOLANGCI_VERSION:=2.1.5
```

Comment:
Can we not upgrade golanci in the same PR. It makes review much more complicated. You don't need to migrate all the tools at once.